docs/fundamentals/code-analysis/quality-rules/ca1000.md

```diff
@@ -59,15 +59,15 @@ To fix a violation of this rule, remove the static member or change it to an ins
 
 Do not suppress a warning from this rule. Providing generics in a syntax that is easy to understand and use reduces the time that is required to learn and increases the adoption rate of new libraries.
 
-## Configurability
+## Configure code to analyze
```

Comment:
The link in line 30 in this article is now broken. That's for nearly all other articles as well.

A quick find & replace in quality-rules folder to replace `#configurability` with `#configure-code-to-analyze` may work well.

Reply:
Looks like I missed quite a lot.